apps/src/templates/Notification.jsx

```diff
@@ -8,37 +8,37 @@ const NotificationType = {
   information: 'information',
   success: 'success',
   failure: 'failure',
-  warning: 'warning'
+  warning: 'warning',
+  course: 'course'
 };
 
 const styles = {
   main: {
     borderWidth: 1,
     borderStyle: 'solid',
-    borderRadius: 3,
```

Comment:
If the Notification styles are drastically different, then, yes, parameters might be the way to go. Is it different enough to be its own component? Or similar enough that the spec could conform to these styles?